src/components/views/rooms/ReadReceiptGroup.tsx

```diff
@@ -201,7 +201,7 @@ function ReadReceiptPerson({ userId, roomMember, ts, isTwelveHour, onAfterClick
         <MenuItem
             className="mx_ReadReceiptGroup_person"
             onClick={() => {
-                dis.dispatch({
+                roomMember && dis.dispatch({
```

Comment:
this remaining a button without a clickable function will be bad for accessibility

maybe you should mimic other callers

```

                // XXX: We should be using a real member object and not assuming what the receiver wants.
                member: member || { userId } as User,
```

Reply:
Oh, that’s good to know! I was wondering whether that’d be possible, thanks!